control/settings/controllers/settings.home.controller.js

```diff
@@ -92,5 +118,12 @@
         $scope.$watch(function () {
           return SettingsHome.couponInfo;
         }, saveDataWithDelay, true);
+
+        // $scope.$watch('SettingsHome.couponInfo.data.settings.employeeCode', function (newVal, oldVal) {
```

Comment:
if not needed lets remove comments please

Reply:
I will remove all commented code that is not needed